UploadServiceLogo/index.ts

```diff
@@ -0,0 +1,77 @@
+import { Context } from "@azure/functions";
+
+import * as express from "express";
+import * as winston from "winston";
+
+import { DocumentClient as DocumentDBClient } from "documentdb";
+
+import {
+  SERVICE_COLLECTION_NAME,
+  ServiceModel
+} from "io-functions-commons/dist/src/models/service";
+import {
+  TelemetryClient,
+  wrapCustomTelemetryClient
+} from "io-functions-commons/dist/src/utils/application_insights";
+import * as documentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+
+import { UploadServiceLogo } from "./handler";
+
+// Whether we're in a production environment
+const isProduction = process.env.NODE_ENV === "production";
+
+const getCustomTelemetryClient = wrapCustomTelemetryClient(
+  isProduction,
+  new TelemetryClient()
+);
+
+const cosmosDbUri = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_URI");
+const cosmosDbKey = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_KEY");
+const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
+const logosHost = getRequiredStringEnv("LOGOS_HOST");
```

Comment:
```suggestion
const logosHost = getRequiredStringEnv("SERVICE_LOGOS_HOST");
```

Reply:
we must document this variable *and* the logos storage connection string into the readme (I know that it is empty atm)